chalicelib/dcc/transformer_mapping.json

```diff
@@ -7,47 +7,28 @@
       },
       {
         "source": "metadata.json",
-        "index_field": "analysis_type",
-        "dss_field": "analysis_type"
+        "index_field": "center_name",
+        "dss_field": "core_metadata.contributor"
       },
       {
         "source": "metadata.json",
-        "dss_field": "center_name",
-        "index_field": "center_name"
+        "dss_field": "project_id",
+        "index_field": "core_metadata.project"
       },
       {
-        "source": "metadata.json",
-        "dss_field": "donor_uuid",
```

Comment:
I expect `redwoodDonorUUID` will likely change to something else or go away at some point, yet for now it is needed for the dashboard service to work.  For the time being, let's use the constant mapping to set these to an empty string.